donate/payments/tasks.py

```diff
@@ -26,6 +27,95 @@
 stripe.api_key = settings.STRIPE_API_KEY
 
 
+def mofo_donation_receipt_datetime(ts):
```

Comment:
lines 30-117 taken from Baskets code, these are all used to process receipt/ send email